app/src/main/java/com/stevesoltys/seedvault/metadata/MetadataManager.kt

```diff
@@ -198,6 +198,8 @@ class MetadataManager(
 
     @Synchronized
     fun getPackagesNumBackedUp(): Int {
+        // FIXME we are under-reporting packages here,
+        //  because we have no way to also include upgraded system apps
```

Comment:
spacing

Reply:
The spacing looks indeed odd, but it is intentional. This way the IDE highlights the multi line comment as FIXME